src/qibolab/platform.py

```diff
@@ -20,39 +20,19 @@ def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=
         cloud (bool): See :class:`qibolab.instruments.qmsim.QMSim` for details.
             Relevant only when ``simulation_duration`` is given.
     """
-    # Instantiate QM OPX instruments
-    if simulation_duration is None:
-        from qibolab.instruments.qm import QMOPX
-        from qibolab.instruments.rohde_schwarz import SGS100A
-
-        controller = QMOPX("qmopx", "192.168.0.1:80")
-
-        # Instantiate local oscillators (HARDCODED)
-        local_oscillators = [
-            SGS100A("lo_readout_a", "192.168.0.39"),
-            SGS100A("lo_readout_b", "192.168.0.31"),
-            SGS100A("lo_drive_low", "192.168.0.32"),
-            SGS100A("lo_drive_mid", "192.168.0.33"),
-            SGS100A("lo_drive_high", "192.168.0.34"),
-            SGS100A("twpa_a", "192.168.0.35"),
-        ]
-        design = MixerInstrumentDesign(controller, local_oscillators)
-
-    else:
-        from qibolab.instruments.qmsim import QMSim
-
-        if address is None:
-            # connect to TII instruments for simulation
-            address = "192.168.0.1:80"
-
-        controller = QMSim("qmopx", address, simulation_duration, cloud)
-        # avoid connecting to local oscillators when simulation is used
-        local_oscillators = []
-        design = BasicInstrumentDesign(controller)
+    # Create channel objects
+    # readout
+    channels = {"L3-25_a": Channel("L3-25_a"), "L3-25_b": Channel("L3-25_b")}
```

Comment:
Maybe we can make a helper function to treat this very common use case:
```py
def channels_from_names(*names: List[str]) -> Dict[str, Channel]:
    return {name: Channel(name) for name in names}
```
and then you could write:
```suggestion
    channels = channels_from_names(["L3-25_a", "L3-25_b"])
```

Or also:
```py
from typing_extensions import Unpack

def channels_from_names(*names: Unpack[str]) -> Dict[str, Channel]:
    return {name: Channel(name) for name in names}
```
(in this last example the type hint is only for clarity, but not being in stdlib up to py3.11 I would actually omit)

The function name is almost random, we can think about a better one (especially shorter, since you might want to use it often).